src/pages/resultsView/ResultsViewPageStoreUtils.ts

```diff
@@ -716,6 +716,27 @@ export function getMultipleGeneResultKey(
         : _.map(groupedOql.list, data => data.gene).join(' / ');
 }
 
+export function calculateQValuesAndSortEnrichmentData<
+    T extends { pValue: number; qValue?: number }
+>(data: T[], sortFunction: (data: any[]) => any[]): any[] {
+    const dataWithpValue: T[] = [];
+    const dataWithoutpValue: T[] = [];
+    data.forEach(datum => {
+        datum.pValue === undefined
```

Comment:
i wouldn't use turnary this way.  it's confusing.  just use if/else.

Reply:
Sure